browserassets/html/admin/hotkeys.html

```diff
@@ -87,7 +87,7 @@ <h3>Emotes</h3>
 			<li><b>Ctrl+L:</b> Laugh</li>
 			<li><b>Ctrl+N:</b> Nod</li>
 			<li><b>Ctrl+Q:</b> Wave</li>
-			<li><b>Ctrl+R:</b> Flip</li>
```

Comment:
what happened to my poor flip

Reply:
I should pay more attention to the Git Graph...